### ISSUE TITLE
Wrong command to activate the environment.

### ISSUE THREAD

**Issue by ARKA1112** [intel/certified-developer#11]
Chapter:  Basics of API and Endpoint Design: 
-------------------

Section:
Lab1:Build REST API Endpoints with FastAPI 
.........................

The HTML document provided in the syllabus section,provides guidelines to create a conda evironment using the requirements provided in the conda.yml file.

While following the steps I found out that it instructs to activate the environment using the command `conda env create -f conda.yml`, instead which should be `conda activate lab1` because it creates an environment named lab1.
![Screenshot from 2022-09-26 13-21-366](https://github.com/intel/certified-developer/assets/24940818/61df604b-cbfe-4346-b0d8-2a786aa7d564)

Thanks.

**Comment by eduand-alvarez** [comment]
This is correct; it must have been an editing error when copying over the images. Thank you for pointing this out.

**Comment by eduand-alvarez** [comment]
I will close issue once it has been updated on the course website.

**Comment by eduand-alvarez** [comment]
The issue has been addressed - closing issue.
